public/js/components/Menu.jsx

```diff
@@ -46,6 +47,9 @@ var Menu = React.createClass({
                       unleash admin
                    </span>
                   </a>
+                  <div style={{position: "absolute", right: "0"}}>
```

Comment:
I tried just that, but the "menu" container is a bit strict. 

You can put it on the same level as the "page" container, but then it will end up on a line below the "blue" bar. 

Any suggestion on how to do this better are welcomed !! =)